### PR TITLE
Responsive register page

### LINKED ISSUE #50: Responsive register page

### PR DESCRIPTION
**Pull request by zeetec20**
Fix #50

### CODE REVIEW

File: app/(auth)/register/page.tsx [hunk 1]
```diff
@@ -120,11 +120,11 @@ const RegisterPage = () => {
...
             </p>
           </div>
-          <div className="mt-1 flex w-full gap-6">
-            <div className="basis-1/2 pl-6">
+          <div className="mt-1 grid grid-cols-2 gap-6">
+            <div className="pl-6">
               <Label htmlFor="country">Country</Label>
               <SelectInput
-                placeholder="Chose Country"
+                placeholder="Chose Country efewf ewfewfwe fewf"
```
Comment (kalwabed) [inline app/(auth)/register/page.tsx:127]: What is "efewf ewfewfwe fewf"?

File: src/components/ui/form/select/select.tsx [hunk 2]
```diff
@@ -47,8 +48,8 @@ const SelectInput = ({
           </span>
         </Listbox.Button>
         <Transition as={Fragment} leave="transition ease-in duration-100" leaveFrom="opacity-100" leaveTo="opacity-0">
-          <Listbox.Options className="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
-            {data.map(e => item(e, iconCheck))}
+          <Listbox.Options className="absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none max-sm:w-1/2 max-xs:w-9/12 sm:text-sm">
```
Comment (kalwabed) [inline src/components/ui/form/select/select.tsx:51]: What are `max-xs` and `max-sm`? Is some custom Tailwind styles?

**Review by kalwabed — CHANGES_REQUESTED** [review]

File: src/components/ui/form/select/select.tsx (hunk 2 above)
Comment (zeetec20) [reply src/components/ui/form/select/select.tsx:51]: it's one of feature tailwind for design responsive, [documentation](https://tailwindcss.com/docs/responsive-design#targeting-a-breakpoint-range)

Comment (kalwabed) [reply src/components/ui/form/select/select.tsx:51]: I see. I just found out.

**Review by kalwabed — APPROVED** [review]
LGTM :100:

**Pull request merged**